network/librewolf/librewolf.info

```diff
@@ -1,10 +1,10 @@
 PRGNAM="librewolf"
-VERSION="109.0.1"
+VERSION="110.0"
 HOMEPAGE="https://librewolf.net/"
 DOWNLOAD="UNSUPPORTED"
 MD5SUM=""
-DOWNLOAD_x86_64="https://gitlab.com/api/v4/projects/24386000/packages/generic/librewolf/$VERSION-1/LibreWolf.x86_64.AppImage"
-MD5SUM_x86_64="497c3f3ba79510be20c58c4d187c442d"
+DOWNLOAD_x86_64="https://gitlab.com/api/v4/projects/24386000/packages/generic/librewolf/$VERSION-2/LibreWolf.x86_64.AppImage"
```

Comment:
here ^ `$VERSION` needs to be the actual version

Reply:
I have submitted several Slackbuilds with a $VERSION in the download link.
Also, `sbolint` does not complain about it.

Should I still update it?

Reply:
Also, my previous update (about a week ago) of that same Slackbuild is using this variable.

Reply:
> Should I still update it?

Yes, it needs to be changed. While info files are valid shell code, not all tools process them that way. Indeed, the components for the sbo db do not. The link for your current submission is broken:

https://slackbuilds.org/repository/15.0/network/librewolf/

> Also, my previous update (about a week ago) of that same Slackbuild is using this variable.

That one should not have made it through